Dockerfiles/cluster-agent/README.md

```diff
@@ -99,24 +55,52 @@ spec:
 Apply the manifest:
 `kubectl create -f dca_deploy.yaml`
 
-### Command line interface of the Cluster Agent
+### Communication with the Datadog Node Agent
+<a name="communication-with-the-datadog-node-agent"></a>
+For the Datadog Cluster Agent to communicate with the Node Agent, you need to share an authentication token between the two agents.
+The token needs to be greater or equal to 32 characters and should only have upper case or lower case letters and numbers.
+You can pass the token as an environment variable: `DD_CLUSTER_AGENT_AUTH_TOKEN`.
+Besides this token, you need to set the `DD_CLUSTER_AGENT_ENABLED=true` in the manifest of the Datadog Node Agent.
+
+## Running the Datadog Cluster Agent with Kubernetes
+
+### Security premise
+<a name="security-premise"></a>
+
+We strongly recommend using a secret to authenticate communication between Agents with the Datadog Cluster Agent.
+You must modify the value of the secret in [the dca-secret.yaml](/manifests/cluster-agent/dca-secret.yaml) then create it:
+
+`kubectl create -f manifests/cluster-agent/dca-secret.yaml`
+
+yields:
+
+```
+kubectl get secret datadog-auth-token
+NAME                 TYPE      DATA      AGE
+datadog-auth-token   Opaque    1         16s
+
+```
+
+## Migration path
 
-The available commands for the Cluster Agents are:
+If you are running the Datadog Node Agent 6.4.2+, to deploy the Datadog Cluster Agent you need to:
+- Deploy [the datadog-cluster-agent_service.yaml](/manifests/cluster-agent/datadog-cluster-agent_service.yaml)
+- Configure and create [the dca-secret.yaml](/manifests/cluster-agent/dca-secret.yaml)
+- Configure (DD_API_KEY and other options) the [the cluster-agent.yaml](/manifests/cluster-agent/cluster-agent.yaml) and deploy it
+- Specify the [required options](#communication-with-the-datadog-node-agent) to ensure the communication between the Datadog Cluster Agent and the Datadog Node Agent.
+- Apply the new configuration to the Datadog Node Agent DaemonSet and redeploy the DaemonSet.
+
```

Comment:
It might be worth adding that you can run `agent status` after these steps to confirm that the Agent has successfully connected to the DCA.